aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Merger.kt

```diff
@@ -47,15 +47,17 @@ internal class Merger(
     private val localStorageAdapter: LocalStorageAdapter
 ) {
     /**
-     * Merge an item back into the local store, using a default strategy.
-     * @param modelWithMetadata A model, combined with metadata about it
+     * Merge a batch of items back into the local store, using a default strategy.
+     * @param modelWithMetadata Models, combined with metadata about it
      * @param <T> Type of model
      * @return A completable operation to merge the model
      </T> */
     fun <T : Model> merge(modelWithMetadata: ModelWithMetadata<T>): Completable {
         return merge(listOf(modelWithMetadata), NoOpConsumer.create())
     }
-
+    fun <T : Model> merge(modelWithMetadata: List<ModelWithMetadata<T>>): Completable {
```

Comment:
remove this new method since it is only used in testing

Reply:
Fixed!